docs/grammar/SolidityParser.g4

```diff
@@ -341,11 +341,10 @@ expression:
 	expression LBrack index=expression? RBrack # IndexAccess
 	| expression LBrack start=expression? Colon end=expression? RBrack # IndexRangeAccess
 	| expression Period (identifier | Address) # MemberAccess
-	| expression LBrace (namedArgument (Comma namedArgument)*)? RBrace # FunctionCallOptions
-	| expression callArgumentList # FunctionCall
+	| expression functionCallOption? callArgumentList # FunctionCall
 	| Payable callArgumentList # PayableConversion
 	| Type LParen typeName RParen # MetaType
-	| (Inc | Dec | Not | BitNot | Delete | Sub) expression # UnaryPrefixOperation
```

Comment:
`Inc` is `++` - what you wanted to disable is `+`, isn't it?

Reply:
Ah, you're right. I'm currently writing a parser per major compiler versions (0.4.0, 0.5.0, etc.) and did not think to check `Inc`. Will revert this.